examples/solids/elasticity.h

```diff
@@ -64,6 +96,7 @@ static const char *const problemTypesForDisp[] = {"Linear elasticity",
                                                   "Hyperelasticity finite strain Initial config Neo-Hookean w/ dXref_dxinit, Grad(u), C_inv, constant storage",
                                                   "Hyperelasticity finite strain Current config Neo-Hookean w/ dXref_dxinit, Grad(u) storage",
                                                   "Hyperelasticity finite strain Current config Neo-Hookean w/ dXref_dxcurr, tau, constant storage",
+                                                  "Hyperelasticity finite strain Initial config coupled Moony-Rivlin w/ dXref_dxinit, Grad(u) storage"
```

Comment:
Do we need to write "coupled" here? Since other NH also are coupled form and we didn't write coupled.